fontforge/parsettf.c

```diff
@@ -5236,6 +5243,7 @@ static void readttfpostnames(FILE *ttf,struct ttfinfo *info) {
 		nm[j] = '\0';
 		if ( indexes[i]<info->glyph_cnt && info->chars[indexes[i]]!=NULL )
 		    info->chars[indexes[i]]->name = nm; /* Too many fonts have badly named glyphs to deduce encoding from name */
```

Comment:
While "we're here" (and since this change supports reading many more strings than there are glyphs), wouldn't an "`else free(nm);` be good here?

Reply:
Makes sense.